i18n/en-US.properties

```diff
@@ -1002,6 +1002,64 @@ boxui.searchForm.searchLabel = Search query
 boxui.securityCloudGame.instructions = For security purposes, please drag the white cloud into the dark cloud.
 # Success message shown when a user successfully drags the cloud into position.
 boxui.securityCloudGame.success = Success!
+# Bullet point that summarizes application download blocked restriction applied to classification
+boxui.securityControls.appDownloadBlock = Some application restrictions apply.
+# Bullet point that summarizes application download restriction applied to classification
+boxui.securityControls.appDownloadList = Some applications will be restricted; {appNames}
+# Bullet point that summarizes application download restriction applied to classification. This variation is used when the list of applications is longer than the configured threshold
+boxui.securityControls.appDownloadListOverflow = Only select applications are allowed; {appNames} +{remainingAppCount} more
```

Comment:
; expected?

Reply:
some places we use `app`, other places `applications`. Should one make them consistent?

Reply:
Yes, the semicolon here is deliberate. I think it was suggested as a grammar improvement.

I'm now using `app` consistently in the code to keep the variable names short, but I will confirm with design if we want to update the copies. I think some of these were optimized for length, but I'll update them in the next PR if they prefer to keep it consistent.